src/overlays/gamestates/ovl_file_choose/z_file_nameset.c

```diff
@@ -87,13 +87,19 @@ void FileSelect_SetKeyboardVtx(GameState* thisx) {
     }
 }
 
-static void* sNameLabelTextures[] = { gFileSelNameENGTex, gFileSelNameENGTex, gFileSelNameFRATex };
+static void* sNameLabelTextures[] =
+    LANGUAGE_ARRAY(gFileSelNameJPNTex, gFileSelNameENGTex, gFileSelNameENGTex, gFileSelNameFRATex);
 
+#if OOT_NTSC
+// TODO: fix NTSC data
```

Comment:
can you elaborate on what is there to fix

Reply:
It should really be
```c
#if OOT_NTSC
static void* sButtonTextures[] = {
    gFileSelHiraganaButtonTex,  gFileSelKatakanaButtonTex, gFileSelKanjiButtonTex,
    gFileSelBackspaceButtonTex, gFileSelENDButtonENGTex,
};

static u16 sButtonWidths[] = { 44, 44, 28, 28, 44 };
#else
static void* sButtonTextures[][2] = {
    { gFileSelBackspaceButtonTex, gFileSelENDButtonENGTex },
    { gFileSelBackspaceButtonTex, gFileSelENDButtonGERTex },
    { gFileSelBackspaceButtonTex, gFileSelENDButtonFRATex },
};

static u16 sButtonWidths[] = { 28, 44 };
#endif
```
it requires different code though and I haven't fully cleaned that up yet, so I just tried to make it build for now

Reply:
maybe
```suggestion
// TODO: some data is missing here for NTSC, as well as related code below
```
then?

Reply:
I went ahead and put the correct data here and moved the comment to the code below (since that's fake right now anyway).